2022/2xxx/CVE-2022-2701.json

```diff
@@ -4,14 +4,72 @@
     "data_version": "4.0",
```

Comment:
'ttps://github.com/E1CHO/water_cve/blob/main/E-learning%20System%20personal%20data%20modification%20XSS%20vulnerability.pdf' does not match '^(ftp|http)s?://\\S+$'